### PR TITLE
[tests-only] Added files tests from testplan

### PR DESCRIPTION
**Pull request by Talank**
### Related Issue
part of https://github.com/owncloud/client/issues/9396

### Test automated
The following test from the testplan has been automated in this PR
```
Test scenario: User adds the various types of files 
Steps to reproduce:
1. Microsoft word documents, Microsoft Excel, Microsoft Powerpoint, .JPG, .PDF, .MP3
Result: The Sync files tab show all type of files added in the Desktop Client folder from desktop, All files snyced and shown in Activities Tab under synchronization protocol
```

I have automated test for syncing files from client to server and vice-versa

### CODE REVIEW

**Comment by Talank** [comment]
Can not create a file (in the GUI test). The `not synced` tab says that file was changed during synced. So, this PR is blocked until https://github.com/owncloud/client/issues/9281#issuecomment-1002033359 is fixed.

**Comment by Talank** [comment]
the issue #9281 has been closed as it has been splitted into specific issues. Now this PR is blocked until #9325  is fixed

**Comment by Talank** [comment]
The blocking issues has been fixed.

**Comment by Talank** [comment]
Files for upload issue is still not resolved in client. So the step `And user "Alice" has uploaded file "testavatar.png" to "simple-folder/testavatar.png" on the server` fails. We first need to either bump the middleware commit or use (and solve files for upload issue in) middleware's docker image.

File: test/gui/tst_syncing/test.feature [hunk 1]
```diff
@@ -292,3 +292,41 @@ Feature: Syncing files
...
+        And the file "simple-folder/testavatar.jpg" should exist on the file system
+        And the file "simple-folder/testavatar.jpeg" should exist on the file system
+        And the file "simple-folder/testimage.mp3" should exist on the file system
+        And the file "simple-folder/test_video.mp4" should exist on the file system
+        And the file "simple-folder/simple.pdf" should exist on the file system
+
+
+    Scenario: various types of files can be synced from client to server
+        Given user "Alice" has set up a client with default settings
+        And user "Alice" has created the following files on the client:
```
Comment (saw-jan) [inline test/gui/tst_syncing/test.feature:318]: for consistency with other similar steps
```suggestion
        And user "Alice" has created the following files inside the sync folder:
```

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by individual-it — APPROVED** [review]

**Pull request merged**